src/lazy/encoder/binary/v1_1/value_writer.rs

```diff
@@ -313,8 +313,8 @@ impl<'value, 'top> BinaryValueWriter_1_1<'value, 'top> {
         // Compute the offset, its width in bits, and how that will affect the opcode and encoded length.
         let (num_offset_bits, offset_value, opcode_adjustment, length_adjustment) =
             match value.offset() {
-                None => (1, 1, 0, 0), // Unknown offset uses a single bit (1); opcode and length stay the same.
-                Some(0) => (1, 0, 0, 0), // UTC uses a single bit (0); opcode and length stay the same.
+                None => (1, 0, 0, 0), // Unknown offset uses a single bit (0); opcode and length stay the same.
+                Some(0) => (1, 1, 0, 0), // UTC uses a single bit (1); opcode and length stay the same.
```

Comment:
🗺️ We only recently defined which `offset` bit value meant `UTC` and which meant `unknown`. I implemented this code long ago and guessed wrong. 🙃